api/Makefile

```diff
@@ -85,6 +85,11 @@ shell:
 	# Run shell in virtual environment shell.
 	poetry shell
 
+copy-prod-db:
+	# Download the production database copy
+	# Make sure to login to Openshift first
+	PROJECT=auzhsi-prod POD=patroni-wps-api-prod-2 DATABASE=wps-api-prod scripts/copy_db_from_pod_to_local.sh
```

Comment:
pg_dump does an explicit lock on the table it's currently working on, so it's best we ensure that we're running pg_dump on a replica, not a leader. If you hard code the pod, you may inadvertently lock the leader database. I'd prefer we not have this shortcut, unless it's smart enough to first identify a replica.

Reply:
Okay, I can remove this. So from now on, I need to check the pod is a replica first then run the command right?

Reply:
Yep - that's best. It will still block updating of the database on that server, but it shouldn't block the leader I don't think.